src/sfdt/__tests__/fixtures/emptyBlockSection.js

```diff
@@ -0,0 +1,770 @@
+module.exports = {
```

Comment:
@kirannbhatt do you need a big sfdt like that ?
Can't we trim it a bit

Reply:
Since toggleBookmark and populate requires whole sfdt to process, we need the complete doc sfdt.

Reply:
Can't we have a valid sfdt structure with less text and so on?

Reply:
Can't we have a valid sfdt structure with less text and so on?

Reply:
I've another one with around 440 lines. Let me add that.

Reply:
updated.